samples/react-native/src/App.tsx

```diff
@@ -277,6 +277,11 @@ function App() {
     <>
       <RootNavigationContainer />
       <RunningIndicator />
+      <Text>
+        <Text>
+          Hello
```

Comment:
q: Do we want the `Hello` text at the bottom in the sample or is it just for testing?

<details><summary>Screenshot</summary>

![Screenshot_20241125_155719](https://github.com/user-attachments/assets/011168ac-7d97-41de-bc22-24033509b2db)
</details>

Reply:
God catch, thank you, now I'll remove it.